package.json

```diff
@@ -292,8 +292,6 @@
 	},
 	"resolutions": {
 		"react-dates": "17.1.1",
-		"@automattic/newspack-blocks/swiper@4.5.1": "patch:swiper@4.5.1#.patches/swiper@4.5.1.diff",
```

Comment:
This diff was removing a self-promotional message from the package which doesn't exist in the new version, so the patch isn't needed.